distribution/build.gradle

```diff
@@ -287,21 +287,21 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
       copySpec {
         // delay by using closures, since they have not yet been configured, so no jar task exists yet
         from { project(':server').jar }
-        from { project(':server').configurations.runtime }
```

Comment:
This is actually necessary as the previously used `runtime` configuration extends compile and is therefore now empty. Using the correct `runtimeClasspath` configuration here has also the nice effect of removing another deprecated behaviour (of using "runtime")